.github/workflows/build.yml

```
@@ -1,14 +1,13 @@
 name: Build
-on: [push, pull_request]
+on:
+  pull_request:
+  push:
+    branches:
+      - main
 
 jobs:
   test:
-    runs-on: ${{ matrix.os }}
-    strategy:
-      fail-fast: false
-      matrix:
-        node-version: [14.x]
-        os: [ubuntu-latest]
+    runs-on: ubuntu-latest
```

Unfortunately, because of this project's `required` status configuration, we need to go back to the `matrix` build here 😢 

Maybe in the future, we can reach out to @JupiterOne/security to update the project settings.

```suggestion
    runs-on: ${{ matrix.os }}
    strategy:
      fail-fast: false
      matrix:
        node-version: [14.x]
        os: [ubuntu-latest]
```

---

yes, will update.